package.json

```diff
@@ -48,5 +49,13 @@
     "simple-html-precompiler": "^1.0.1",
     "stripe": "~2.2.0",
     "webpack": "^1.9.10"
+  },
+  "devDependencies": {
+    "colors": "^1.1.2",
+    "fs-readfile-promise": "^2.0.1",
+    "fs-writefile-promise": "^1.0.1",
+    "js-yaml": "^3.3.1",
+    "xml2json": "^0.7.1",
+    "yaml-intl-xml-json-converter": "0.0.7"
```

Comment:
What are each of these doing?

It's not clear where in the code these are loaded in, I wonder if some of these are from testing?